server/lib/manifest.py

```diff
@@ -23,21 +25,24 @@ class Manifest:
     create<someProperty>
     """
 
-    def __init__(self, tale, user, expand_folders=True, versionId=None):
+    def __init__(self, tale, user, expand_folders=True, is_export=False, version_id=None):
         """
         Initialize the manifest document with base variables
         :param tale: The Tale whose data is being serialized
         :param user: The user requesting the manifest document
         :param expand_folders: If True, when encountering a folder
             in the external data, return all child items recursively.
+        :param is_export: Boolean that signals the manifest is being written/read
```

Comment:
I think this introduces a bit of circular dependency (at least in the logic if not code). Manifest as a class/object shouldn't really care about the fact it's being exported. If it's already written to disk we don't really need to recreate the instance of this class. We could use `ManifestParser(the_file).manifest` as the *thing* we pass to the exporter.